app/models/concerns/route_ten_decisions.rb

```diff
@@ -0,0 +1,15 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module RouteTenDecisions
+  extend ActiveSupport::Concern
+
+  included do
+    has_decision :ten_agency_confirm_match_success, decision_class_name: 'MatchDecisions::Ten::TenAgencyConfirmMatchSuccess', notification_class_name: 'Notifications::Ten::TenAgencyConfirmMatchSuccess'
+    has_decision :ten_agency_confirm_match_success_decline, decision_class_name: 'MatchDecisions::Ten::TenAgencyConfirmMatchSuccessDecline', notification_class_name: 'Notifications::Ten::TenAgencyConfirmMatchSuccessDecline'
+    has_decision :ten_confirm_match_success_dnd_staff, decision_class_name: 'MatchDecisions::Ten::TenConfirmMatchSuccessDndStaff', notification_class_name: 'Notifications::Ten::TenConfirmMatchSuccessDndStaff'
```

Comment:
Maybe this doesn't matter, but I think it might be more clear if we start using `coc_staff` and `CocStaff` instead of `dnd_staff` and `DndStaff`